### PR TITLE
Fix rows reorder when pagination is applied.

### PR DESCRIPTION
**Pull request by sammanan4**
When the pagination is activated and the rows are reordered on pages other than the first one, the order was not taking into account the page number and the page size. This commit changes the index at which the row needs to be inserted by accounting for the page size and page number.

**🤔Type of Request**
- [x] **Bug fix**

**📝Changelog**

- [x] **Extensions**

### CODE REVIEW

File: src/extensions/reorder-rows/bootstrap-table-reorder-rows.js [hunk 1]
```diff
@@ -97,6 +97,8 @@ $.BootstrapTable = class extends $.BootstrapTable {
 
   onDrop (table) {
     this.$draggingTd.css('cursor', '')
+    const pageNum = $(table).bootstrapTable('getOptions').pageNumber;
+    const pageSize = $(table).bootstrapTable('getOptions').pageSize;
```
Comment (wenzhixin) [inline src/extensions/reorder-rows/bootstrap-table-reorder-rows.js:101]:
```suggestion
    const pageNum = this.options.pageNumber
    const pageSize = this.options.pageSize
```

File: src/extensions/reorder-rows/bootstrap-table-reorder-rows.js [hunk 2]
```diff
@@ -109,7 +111,7 @@ $.BootstrapTable = class extends $.BootstrapTable {
     const draggingRow = this.data[this.draggingIndex]
     const droppedIndex = newData.indexOf(this.data[this.draggingIndex])
     const droppedRow = this.data[droppedIndex]
-    const index = this.options.data.indexOf(this.data[droppedIndex])
+    const index = ((pageNum - 1)*pageSize) + this.options.data.indexOf(this.data[droppedIndex])
```
Comment (wenzhixin) [inline src/extensions/reorder-rows/bootstrap-table-reorder-rows.js:114]:
```suggestion
    const index = (pageNum - 1) * pageSize + this.options.data.indexOf(this.data[droppedIndex])
```

**Comment by wenzhixin** [comment]
Please provide an [Online Example](https://live.bootstrap-table.com) to show your problem. Thanks!

**Comment by sammanan4** [comment]
Please see https://examples.bootstrap-table.com/#extensions/reorder-rows.html
When trying to reorder the rows on any page other than the first one, and then change the page, you can see the ordering is not correct. Thanks.

**Comment by UtechtDustin** [comment]
This is an example using you're commit: https://live.bootstrap-table.com/code/UtechtDustin/16248
Please check the console, the reording is broken.
```
Uncaught SyntaxError: redeclaration of const pageNum
note: Previously declared at line 100, column 10
```

**Comment by sammanan4** [comment]
@UtechtDustin I have fixed the redeclaration issue in the previous commit, please try to run again. I am not able to load the updated code in the live editor though.

**Review by UtechtDustin — APPROVED** [review]

**Comment by UtechtDustin** [comment]
Thank you for the contibution, the latest commit is in the cache of the used cdn.   
I tested it locally, works fine.

**Pull request merged**